extensions/binding/org.eclipse.smarthome.binding.yahooweather/src/main/java/org/eclipse/smarthome/binding/yahooweather/handler/YahooWeatherHandler.java

```diff
@@ -36,6 +37,10 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.google.common.cache.Cache;
+import com.google.common.cache.CacheBuilder;
```

Comment:
Ouch! Sooner or later it had to come after us. Different Guava lib versions are incompatible with each other and it seems that the one in openHAB has issues with the CacheBuilder, see https://github.com/openhab/openhab/issues/4987. Not sure how to solve this, though?